packages/api/test/addbuild-getbuild.js

```diff
@@ -0,0 +1,267 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const { describe, it, after, before } = require('mocha');
+const { Firestore } = require('@google-cloud/firestore');
+const assert = require('assert');
+const { v4: uuidv4 } = require('uuid');
+
+const TestCaseRun = require('../lib/testrun');
+const addBuild = require('../src/add-build');
+const fetch = require('node-fetch');
+
+const TESTING_COLLECTION_BASE = 'repositories-testsuite-';
```

Comment:
this approach is going to start to cause problems when the project gets more active, we have a few branches happening at the same time, and potentially a few test runs happening at the same time.

An approach I've seen work well is to use a UUID in the collection name, so something like:

```
const TESTING_COLLECTION_BASE = `repositories-testsuite-${uuidv4}`;
```

You would also need to:

1. delete the `TESTING_COLLECTION_BASE` after the test run.
2. have an additional cleanup step that looks for collections with the name `repositories-testsuite*` older than N hours, and cleans these up.

☝️ mind creating an issue to do this work, I'm fine with it not being part of the suite yet.

Reply:
This was currently being done a few lines down: 
`global.headCollection = TESTING_COLLECTION_BASE + uuidv4();`